lighthouse-core/gather/driver.js

```diff
@@ -1241,7 +1241,7 @@ class Driver {
    * @return {Promise<void>}
    */
   async beginEmulation(settings) {
-    // TODO(phulce): remove this flag on next breaking change
+    // TODO: https://github.com/GoogleChrome/lighthouse/issues/7044
```

Comment:
I'm not sure if it's worth keeping the TODO for things like this that aren't so targeted at a single point (or that single point is easy to find again since the goal is to remove something altogether)

Reply:
removed